package.json

```diff
@@ -1,15 +1,14 @@
 {
-  "name": "@netlify/plugin-angular-universal",
-  "version": "1.0.1",
-  "description": "Netlify Build plugin Angular Universal - Run Angular Universal seamlessly on Netlify.",
+  "name": "@netlify/angular-runtime",
+  "version": "2.0.0",
```

Comment:
release-please should handle the versioning, though you may need to bootstrap it as a new package. I don't think it should be 2.0.0 though, as it's a new package so semver doesn't apply

Reply:
Let's stick with `2.0.0` so that the Git tags in this repo don't clash.

Reply:
If rlewase-please is setup ok then you shouldn't update this here: the breaking change will trigger a major bump anyway